CHANGELOG.md

```diff
@@ -1,6 +1,25 @@
 # Changelog
 
-## [v0.1.0] 
+## 0.2.0 (August 23, 2019)
+
+FEATURES:
+
+* Add support for LBaaS (GH-2)
+* Add support for PaaS (GH-6)
+* Add support for ISO Image Handling (GH-8)
+* Add support for Object Storage (GH-11)
+* Add support for Snapshots (GH-12) and Snapshot Scheduler (GH-13)
+* Add support for Firewall Handling (GH-14)
+
+IMPROVEMENTS:
+
+* Avoid use of fmt.errorf
```

Comment:
We still use `fmt.errorf` in some lines.

Reply:
hehe, then it war re-introduced. because at some point I deliberately removed it ;)